components/sections/pricing/variant_d.tsx

```diff
@@ -1,613 +1,638 @@
-														xmlns="http://www.w3.org/2000/svg"
-														fill="none"
-														viewBox="0 0 24 24"
-														stroke="currentColor">
-														<path
-															strokeLinecap="round"
-															strokeLinejoin="round"
-															strokeWidth={2}
-															d="M16 12a4 4 0 10-8 0 4 4 0 008 0zm0 0v1.5a2.5 2.5 0 005 0V12a9 9 0 10-9 9m4.5-1.206a8.959 8.959 0 01-4.5 1.207"
-														/>
-													</svg>
-												)}
-											</div>
-										)}
-									</div>
-								);
-							})}
-							<div className="mb-5 text-left text-sm text-gray-500">
-								<label className="inline-flex">
-									<input
-										aria-label="Agree to terms"
-										className="mr-2"
-										type="checkbox"
-										name="terms"
-										defaultValue={1}
-									/>
-									<PortableText value={block} components={blockCustomization} />
-								</label>
-							</div>
-							<div>
-								<div className="webriq-recaptcha" />
-							</div>
-							<button
-								id="submitBtn"
-								aria-label="Submit Pricing Form button"
-								onClick={(e) => handleSubmit(e)}
-								// type="submit"
-								// onClick={() =>
-								//   initiateCheckout(
-								//     {
-								//       lineItems: [
-								//         {
-								//           price:
-								//             billing.billType === "Monthly"
-								//               ? useCheckout.monthlyCheckout
-								//               : useCheckout.yearlyCheckout,
-								//           quantity: 1,
-								//         },
-								//       ],
-								//     },
-								//     stripePKey,
-								//     window.location.origin + "/success",
-								//     window.location.href,
-								//     true,
-								//     setPKError
-								//   )
-								// }
-								className={`block w-full rounded-l-xl rounded-t-xl bg-webriq-blue p-4 text-center font-bold leading-none text-white transition duration-200 hover:bg-webriq-darkblue ${
-									billing.billType === "" &&
-									"cursor-not-allowed disabled:opacity-50"
-								}`}
-								disabled={billing.billType === ""}>
-								{/* {paymentOngoing
+    return (
+      <div className="w-full mb-8 md:mb-0 md:w-1/2">
+        <div className="px-6 py-8 text-center lg:px-8">
+          <Text className="mb-8 text-2xl">{formName}</Text>
+          {formFields && (
+            <WebriQForm
+              stripepkey={stripePKey}
+              method="POST"
+              data-form-id={formId}
+              name={formName}
+              className="form-pricing"
+              data-thankyou-url={thankYouPageLink(formThankYouPage)}
+              scriptsrc="https://pagebuilderforms.webriq.com/js/initReactForms"
+            >
+              {formFields?.map((field, index) => {
+                return (
+                  <div key={index}>
+                    {field.pricingType === "textarea" ? (
+                      <div className="mb-4">
+                        <textarea
```

Comment:
Kindly update this to use the TextArea UI component, and the rest of the input fields to use the Input UI.

Reply:
Resolved thanks